R/analyzer.R

```diff
@@ -272,9 +275,25 @@ extract_pauta <- function(agenda, tabela_geral_ids_casa, export_path) {
     dplyr::filter(dplyr::row_number()==dplyr::n()) %>%
     dplyr::ungroup() %>%
     fix_nomes_locais() %>%
-    dplyr::select(-em_pauta)
+    dplyr::select(-em_pauta) %>%
+    dplyr::mutate(id_ext = as.numeric(id_ext),
+                  data = as.character(data))
+
+  #lubridate::force_tz(pautas, tzone = "America/Los_Angeles")
+  hoje <- Sys.Date()
+  semana_atual <- lubridate::epiweek(hoje)
+  semana_retrasada <- semana_atual - 2
```

Comment:
Pra evitar o problema da virada do ano, pode usar a data da segunda-feira que dá início à semana retrasada. Daí você filtra as pautas cuja data é maior ou igual a essa data.
Ex: para a semana atual (23), a segunda-feira é 03/06/2019. Duas semanas atrás seria a semana 21, cuja segunda-feira é 20/05/2019. Assim, das pautas já baixadas, só vamos aproveitar o que vier até antes do dia 20/05/2019.